server/src/graphql/queries/isPluginInstalled.ts

```diff
@@ -0,0 +1,23 @@
+import { QueryResolvers } from '../../generated/graphql';
+import { sshConnect } from '../../lib/ssh';
+import { dokku } from '../../lib/dokku';
+
+export const isPluginInstalled: QueryResolvers['isPluginInstalled'] = async (
+  _,
+  { pluginName },
+  { userId }
+) => {
+  if (!userId) {
+    throw new Error('Unauthorized');
+  }
+
+  const ssh = await sshConnect();
+
+  const dokkuPlugins = await dokku.plugin.list(ssh);
```

Comment:
you can use server/src/lib/dokku/plugin/installed.ts that will already return a boolean, will be faster ;)